www/lendingbot.js

```diff
@@ -370,6 +376,30 @@ function doSave() {
     });
     localStorage.setItem('timespanNames', JSON.stringify(timespanNames))
 
+    // Bitcoin Display Unit
+    localStorage.displayUnitText = $('input[name="btcDisplayUnit"]:checked').val();
```

Comment:
Wow, I didn't realize this worked!

However, just because we don't have `protected` or `private` here, we shouldn't really skip past the getters/setters... http://stackoverflow.com/questions/12632463/is-localstorage-getitemitem-better-than-localstorage-item-or-localstoragei

> *As Christoph points out in his answer, the functional interface is the only way to reliably store and retrieve values under keys equal to the predefined properties of localStorage*

We're setting ourselves up to get shot in the foot, directly accessing these parameters and muddying up `localStorage`. Also, it's inconsistent doing it two ways.